### DISCUSSION TITLE
umi中配置styleProvider没有生效?

### DISCUSSION THREAD

**Discussion by KiraWong** [umijs/umi#11346] [Q&A]
npx create-umi@latest下载Ant Design Pro的模板，然后在.umirc.ts文件中配置
antd: {
    styleProvider: {
      hashPriority: 'high',
      legacyTransformer: true,
    },
  }
编译后ant样式仍然是:where的样式，怎么解决，还需要配置什么吗？

**Comment by fz6m** [comment]
在 #11347 中修复。

目前自己可以先在 `src/app.tsx` 里自己套一层 `<StyleProvider>` 解决，参考 antd [文档](https://ant.design/docs/react/compatible-style-cn#where-%E9%80%89%E6%8B%A9%E5%99%A8)：

```ts
// src/app.tsx
import { StyleProvider } from '@ant-design/cssinjs'

export const rootContainer = (c) => {
  return <StyleProvider hashPriority=“high”>{c}</StyleProvider>
}
```

注意需要自己安装 `@ant-design/cssinjs` 。
